### ISSUE TITLE
Top-Level Comments

### ISSUE THREAD

**Issue by nyushko** [CMPUT301W14T04/Team04Project#14]
Hey Tarek, quick question. So looking at your code last night, are you putting each comment into a different file? ex, .../TestCases/t1; .../TestCases/t100; etc?
Also, how close are you to being done the top level comment code?

**Comment by telbohtimy** [comment]
It is my understanding that is how its supposed to be done. Testcases would be the group and t1 would be its Id. I've done comment code in general I'll get to top level when I finish elastic search

**Comment by nyushko** [comment]
but then is there supposed to be anything at all in the .../TestCases file?

**Comment by telbohtimy** [comment]
No there would be a comment in TestCases/1 and TestCases/2 but if i try to get a testcase and not give it an id i get this message "No handler found for uri [/cmput301w14t04/TestArea/] and method [GET]".  After giving an Id though I get "{"_index":"cmput301w14t04","_type":"TestArea","_id":"1","_version":51,"exists":true, "_source" : {"comment":"hi!","date":"Mar 05, 2014-03:32","user":"nyushko"}}"
So I am assuming that when searching for these you will look by _type then retrieve all Id's

**Comment by nyushko** [comment]
Is the comment posting to the url of the new code you pushed? I can't get it to work.

**Comment by telbohtimy** [comment]
http://cmput301.softwareprocess.es:8080/cmput301w14t04/TestArea/Test-+-(date) you might have to change the code to post it to a different location. i choose to post it to with the date because its easier to find than the id and I just made the date 1,2,3,4.. etc so http://cmput301.softwareprocess.es:8080/cmput301w14t04/TestArea/Test-+-3 has a comment

**Comment by telbohtimy** [comment]
Later on when I get retrieving and deleting i will change the url so it can post by id's

**Comment by nyushko** [comment]
So each comment is still in a different url?

**Comment by telbohtimy** [comment]
I have tried my best to but an arraylist but for some reason it refuses.  I made sure the arraylist got converted to a string an then when i tried to send it i got an http error 400:bad request.  So then I decided to put that arraylist into a class and then upload that class but then when I checked it would upload but it would be empty.  In this case I'm not sure it uploaded at all or maybe it did and elastic search doesn't want to show it.  So in order for retrieving my plan is a little different but for top-level comments we add a new parameter "type" and replies we also add "type" and I am trying to do a search and retrieve with all these things with a specific type on a certain URL and then take those the match the type that I want to find and add them to an arraylist then return that arraylist.  Problem the code is having is that it is not retrieving anything so i have to solve that problem.
